### PR TITLE
Update FLEDGE services timeline

### PR DESCRIPTION
**Pull request by kevinkiklee**
This PR updates the timeline to match the timeline from the Key/Value service announcement post.

### CODE REVIEW

File: site/en/blog/fledge-service-overview/index.md [hunk 1]
```diff
@@ -77,6 +77,8 @@ look-ups and other advanced queries.
 By implementing a cloud service, adtechs can securely store data for their use
 and keep it up-to-date as the ad campaign progresses.
 
+The [FLEDGE Key/Value service](https://github.com/WICG/turtledove/blob/main/FLEDGE_Key_Value_Server_API.md) that is TEE-based has been [open sourced](/blog/open-sourcing-fledge-key-value-service/), and the source code can be found in the [Privacy Sandbox Github Repository](https://github.com/privacysandbox/fledge-key-value-service).
```
Comment (heyawhite) [inline site/en/blog/fledge-service-overview/index.md:80]:
```suggestion
The TEE-based [FLEDGE Key/Value service](https://github.com/WICG/turtledove/blob/main/FLEDGE_Key_Value_Server_API.md) has been [open sourced](/blog/open-sourcing-fledge-key-value-service/).
```

File: site/en/blog/fledge-service-overview/index.md [hunk 2]
```diff
@@ -153,23 +155,15 @@ FLEDGE for Chrome currently allows a
...
 
 #### Timeline
 
-Adtech platforms who have implemented the BYOS model can consider migrating to
-a TEE-based Key/Value service implementation while FLEDGE is still in
-development. 
-
-We aim to have an initial TEE-based version of the Key/Value service available
-for experimentation by the end of 2022.
+Adtech platforms who have implemented the BYOS model can consider migrating to a TEE-based Key/Value service implementation while FLEDGE is still in development. In the long-term, adtechs will need to use the open source FLEDGE Key/Value services running in trusted execution environments (TEEs) for retrieving real-time data. To ensure that the ecosystem has sufficient time to test, we don’t expect to require the use of the open-source Key/Value services or TEEs until sometime after third-party cookie deprecation. We will provide substantial notice for developers to begin testing and adoption before this transition takes place.
```
Comment (heyawhite) [inline site/en/blog/fledge-service-overview/index.md:166]: This was discussed for a long time, and I don't feel comfortable making this change at 
```suggestion
Adtech platforms who have implemented the BYOS model can consider migrating to
a TEE-based Key/Value service implementation while FLEDGE is still in
development.

In the future, adtechs will need to use the open source FLEDGE Key/Value services
running in trusted execution environments (TEEs) to retrieve real-time data.
To ensure that the ecosystem has sufficient time to test, we don’t expect to
require the use of the open-source Key/Value services or TEEs until sometime
after third-party cookie deprecation. We will provide substantial notice for
developers to begin testing and adoption before this transition takes place.
```

File: site/en/blog/fledge-service-overview/index.md [hunk 3]
```diff
@@ -153,23 +155,15 @@ FLEDGE for Chrome currently allows a
 for the Key/Value service, which will need to be migrated to TEEs in the future.
 The BYOS model is not in scope for [Bidding and Auction services](https://github.com/privacysandbox/fledge-docs/blob/main/bidding_auction_services_api.md).
 
-To ease the migration from the BYOS model, we're providing new [open source
-APIs, documentation, server implementation, and explainers](/blog/open-sourcing-fledge-key-value-service/)
-with additional capabilities beyond those already proposed. The APIs intend to
-allow custom scripts and adtech-defined code which can be run on TEEs.
+To ease the migration from the BYOS model, we're providing new [open source APIs, documentation, server implementation, and explainers](/blog/open-sourcing-fledge-key-value-service/) for the FLEDGE Key/Value service with additional capabilities beyond those already proposed. The APIs intend to allow custom scripts and adtech-defined code which can be run on TEEs.
```
Comment (heyawhite) [inline site/en/blog/fledge-service-overview/index.md:158]:
```suggestion
To ease the migration from the BYOS model, we're providing new [open source
APIs, documentation, server implementation, and explainers](/blog/open-sourcing-fledge-key-value-service/)
for the FLEDGE Key/Value service with additional capabilities beyond those
already proposed. These APIs intend to allow custom scripts and
custom code by adtechs, which can be run on TEEs.
```

File: site/en/blog/fledge-service-overview/index.md (hunk 2 above)
Comment (kevinkiklee) [reply site/en/blog/fledge-service-overview/index.md:170]: Synced offline

File: site/en/blog/fledge-service-overview/index.md [hunk 4]
```diff
@@ -153,23 +155,19 @@ FLEDGE for Chrome currently allows a
...
 
 #### Timeline
 
-Adtech platforms who have implemented the BYOS model can consider migrating to
-a TEE-based Key/Value service implementation while FLEDGE is still in
-development. 
-
-We aim to have an initial TEE-based version of the Key/Value service available
-for experimentation by the end of 2022.
+Adtech platforms who have implemented the BYOS model can consider migrating to a TEE-based Key/Value service implementation while FLEDGE is still in development. In the long-term, adtechs will need to use the open source FLEDGE Key/Value services running in trusted execution environments (TEEs) for retrieving real-time data. To ensure that the ecosystem has sufficient time to test, we don’t expect to require the use of the open-source Key/Value services or TEEs until sometime after third-party cookie deprecation. We will provide substantial notice for developers to begin testing and adoption before this transition takes place.
```
Comment (heyawhite) [inline site/en/blog/fledge-service-overview/index.md:170]:
```suggestion
Adtech platforms who have implemented the BYOS model can consider migrating
to a TEE-based Key/Value service implementation while FLEDGE is still in development.

In the long-term, adtechs will need to use the open source FLEDGE
Key/Value services running in trusted execution environments (TEEs)
for retrieving real-time data. To ensure that the ecosystem has sufficient
time to test, we don’t expect to require the use of the open-source
Key/Value services or TEEs until sometime after third-party cookie
deprecation. We will provide substantial notice for developers to begin
testing and adoption before this transition takes place.
```

Comment (heyawhite) [reply site/en/blog/fledge-service-overview/index.md:170]: FYI that's just line breaks for my readability, no copy was changed.

**Review by heyawhite — APPROVED** [review]

**Pull request merged**